app/containers/Facilities/FacilitiesListContainer.tsx

```diff
@@ -0,0 +1,143 @@
+import React, {useEffect} from 'react';
+import {graphql, createRefetchContainer} from 'react-relay';
+import {Table, Container, Row, Col} from 'react-bootstrap';
+import SortableTableHeader from 'components/SortableTableHeader';
+import SearchBox from 'components/SearchBox';
+import FacilitiesRowItemContainer from './FacilitiesRowItemContainer';
+
+export const FacilitiesList = props => {
```

Comment:
SonarCloud is complaining, rightly so, about this component being a duplicate of `ApplicationsList`. The only difference as far as I can tell is how the rows are rendered. To remove that code duplication, we need to create a new reusable component with a `renderRow` prop.

Reply:
So, a new component that will get all that data as props and outputs it in table rows?